docs/UserGuide.md

```diff
@@ -349,54 +377,139 @@ Examples:
 * `findstu kent ridge` returns `Kent Tan` and `Mark Ridge`<br>
 [Insert image of an example of result of `findstu Cena`]
 
-#### 5.3.4. Editing a student's particulars : `editstu`
+{ start of `features#student#deletestu` written by: Marcus Tan Wei }
 
-The user will be prompted to confirm their decision here.
+#### 5.3.4. Deleting a student : `deletestu`
 
-If yes: edits and updates the particulars of the specified student from the student list.
+Removes a student from the student list.
 
-```
-editstu INDEX n/UPDATED_NAME
-```
+<div markdown="block" class="alert alert-info">
 
-* Edits the student at the specified `INDEX` with the specified updated particular to be updated.
+**:information_source: Notes about the usage of this command:**<br>
+
+* Deletes the student at the specified `INDEX`.
 * The index refers to the index number shown in the displayed student list.
 * The index **must be a positive integer** 1, 2, 3, …
 * User will then be prompted for a confirmation input **`(yes/no)`**.
+   * If yes: deletes the specified student from the student list.
+   * If no: student will not be removed from the student list.
 
-Examples:
-* `editstu 2 n/John Cena` followed by `yes` edits the 2nd student in the student list with an updated name `John Cena`.
-* `editstu 3 t/Joker` followed by `y` edits the 3rd student in the student list with an updated tag `Joker`.
+</div>
 
-#### 5.3.5. Deleting a student : `deletestu`
+**Format:**
+```
+deletestu INDEX
+```
 
-The user will be prompted to confirm their decision here.
+**Expected result:**
+```
+Delete INDEX? (yes/no)
+```
 
-If yes: deletes the specified student from the student list.
+:tip: After receiving the confirmation prompt, enter `yes` to confirm the removal of student at the specified `INDEX`, or else, reject the command by entering `no`.
```

Comment:
Perhaps could be better phrased to ":tip: After receiving the confirmation prompt, enter `yes` to confirm the removal of student at the specified `INDEX` or reject the command by entering `no`"?